src/components/upsell/joinChannel.js

```diff
@@ -4,21 +4,24 @@ import { connect } from 'react-redux';
 import compose from 'recompose/compose';
 import toggleChannelSubscriptionMutation from 'shared/graphql/mutations/channel/toggleChannelSubscription';
 import type { ToggleChannelSubscriptionType } from 'shared/graphql/mutations/channel/toggleChannelSubscription';
-import { addToastWithTimeout } from '../../actions/toasts';
+import { addToastWithTimeout } from 'src/actions/toasts';
+import { openModal } from 'src/actions/modals';
 import type { Dispatch } from 'redux';
+import { withCurrentUser } from 'src/components/withCurrentUser';
 import {
   JoinChannelContainer,
   JoinChannelContent,
   JoinChannelTitle,
   JoinChannelSubtitle,
 } from './style';
-import { Button } from '../buttons';
+import { Button } from 'src/components/buttons';
```

Comment:
Hey @brianlovin, I wanna ask why you changed this from the relative to the absolute path. I've noticed this pattern `src/components/..` throughout the codebase and just wanna know if there's any reason?

Reply:
Partially readability, partially aesthetics (e.g. trying to interpret `../../../utils.js` is much harder to parse than `api/utils.js`).

Reply:
Ah alright makes sense. Thanks